src/tests/validity/variable_ref_operators.ab

```diff
@@ -0,0 +1,16 @@
+// Output
+// 0
```

Comment:
Your other three test files make sense, but I'm not sure what you're trying to accomplish with this one.

Reply:
Surely with:
```
var %= 1
echo var
```
It's going to output "0" for *any* integer input?

Reply:
I asked for that test infact as it wasn't clear to me.

If I try `let var += 1` there is a error from amber but we have another ticket for implement that kind of test errors.

Reply:
I think just to check if those operators works, https://github.com/amber-lang/amber/issues/433

Reply:
Ok understood, but this test does *not* confirm that the shorthand operators work as expected.  Suppose `var` is 4 as in your first example. If `var += 1` were to replace 4 with (say) 999 instead of 5, the `echo` will still output 0 and pass, because the last operation you're doing is a modulo, and any integer modulo 1 is 0.

We already have tests for the shorthand operators:

```
$ cat shorthand_add.ab
// Output
// 64

let x = 16
x += 48
echo x
```

If you want a test for shorthand operators *passed by reference to a function*, I suggest cloning the existing ones:

```
$ cat variable_ref_shorthand_add.ab
// Output
// 64

fun update(ref y) {
    y += 48
}

let x = 16
update(x)
echo x
```

Reply:
yeah it was my doubt that we have that stuff already covered by tests.